api-gateway/src/main/resources/application.yml

```diff
@@ -79,6 +82,10 @@ spring:
           filters:
             # If SESSION cookie is passed to downstream, it is then removed, because downstream discards it
             - RemoveRequestHeader=Cookie
+        - id: neo4j_route
+          uri: ${gateway.neo4j-browser.url}
+           predicates:
+             - Path=/neo4j/browser/**
```

Comment:
maybe add
```
filters:
            # If SESSION cookie is passed to downstream, it is then removed, because downstream discards it
            - RemoveRequestHeader=Cookie
```
?